### ISSUE TITLE
Reaction 'Subject Template' field not updating

### ISSUE THREAD

**Issue by hdaddeo** [meshifyiot/carbon-docs#195]
Hey guys,
In hsbrd.carbon.meshify.com, I am trying to update the 'Subject Template' field. I change the value in the field, and click save.  I receive the 'Updated reaction' notification.
However, when I go back into the reaction, the value is unchanged.  This problem does not seem to occur with API's, only with the UI.
Thanks!

**Comment by decker405** [comment]
We're looking into it today.

**Comment by hdaddeo** [comment]
Hey guys,
I went to update the subject template field and it still didn't stick.  Also, when adding creating a sms notification, it says that the subject template field is not valid. I then changed the reaction type to 'email,' put 'asdf' in the subject template field, changed the reaction type back to 'sms,' and was able to save.
Just letting you know.  Still having no problems with updating via the API.
